src/status_im/contexts/wallet/wallet_connect/responding_events.cljs

```diff
@@ -114,8 +114,9 @@
 
 (rf/reg-event-fx
  :wallet-connect/send-response
- (fn [{:keys [db]} [{:keys [result error]}]]
-   (when-let [{:keys [id topic] :as event} (get-in db [:wallet-connect/current-request :event])]
+ (fn [{:keys [db]} [{:keys [request result error]}]]
+   (when-let [{:keys [id topic] :as event} (or request
+                                               (get-in db [:wallet-connect/current-request :event]))]
```

Comment:
Is `request` always available for this function?

If so, then the `when-let` could just be a single let. Otherwise, could you please share in which cases this function returns `nil` sue to `when-let`?

Reply:
for this i'm not aware about all possible scenario and it was there already, but either request should be available or should be something in db. maybe with when-let is safer, no?

Reply:
It was there because, I guess, it was possible that the info in `db` was not set, but now we are also receiving `request` as parameter so I thought at least one of those would always be passed.

If we are unsure, then we can keep `when-let` 👍